keras_nlp/layers/modeling/f_net_encoder_test.py

```diff
@@ -78,45 +80,14 @@ def test_one_training_step_of_f_net_encoder(self):
         x = keras.layers.Dense(1, activation="sigmoid")(x)
         model = keras.Model(inputs=inputs, outputs=x)
 
-        data = tf.random.uniform(shape=[2, 4, 6])
-        label = tf.cast(data[:, :, 0] >= 0.5, dtype="int32")
+        data = ops.random.uniform(shape=[2, 4, 6])
+        label = ops.random.randint(minval=0, maxval=2, shape=(2, 4, 1))
 
-        loss_fn = keras.losses.BinaryCrossentropy(from_logits=False)
+        loss = keras.losses.BinaryCrossentropy(from_logits=False)
         optimizer = keras.optimizers.Adam()
-        with tf.GradientTape() as tape:
-            pred = model(data)
-            loss = loss_fn(label, pred)
-        grad = tape.gradient(loss, model.trainable_variables)
-        self.assertGreater(len(grad), 1)
-        optimizer.apply_gradients(zip(grad, model.trainable_variables))
-
-    def test_checkpointing_f_net_encoder(self):
```

Comment:
Is this still useful as a `tf_only` test?

Reply:
IDK, are you testing any checkpoint stuff in KerasCV? This is kind of just a weird holdover we don't do consistently.

We have so many serialization things we could check, I just thought this was a little much. Better to focus on Keras saving/loading.

But we definitely could keep testing checkpoints if we want?

Reply:
talked offline. let's ditch these